src/v1.js

```diff
@@ -136,19 +138,27 @@ api.declare({
   ].join('\n'),
 }, async function(req, res) {
   let prefix = req.query.prefix;
+  let continuationToken  = req.query.continuationToken || undefined;
+  let limit         = parseInt(req.query.limit || 10, 10);
+  let Client = this.Client;
+  let resolver = this.resolver;
 
-  // Load all clients
-  // TODO: as we acquire more clients, perform the prefix filtering in Azure
-  let clients = [];
-  await this.Client.scan({}, {
-    handler: client => {
+  let response = {clients: [], continuationToken};
+
+  while (true) {
+    let data = await Client.scan({}, {limit, continuation: response.continuationToken});
```

Comment:
Let's use `continuationToken` here (the variable, rather than the property of `response`).